src/spikeinterface/widgets/gtstudy.py

```diff
@@ -140,16 +134,18 @@ class StudyPerformances(BaseWidget):
     study: GroundTruthStudy
         A study object.
     mode: str
-        Which mode in "swarm"
+        Which mode in "ordered", "snr", "swarm"
```

Comment:
I don't know what 'swarm' is... Is this common knowledge or would it be beneficial to define this? How would a user know when to use 'ordered' vs 'snr' vs 'swarm'....

Reply:
@zm711 I think it's quite common, but we can ref the seaborn page: https://seaborn.pydata.org/generated/seaborn.swarmplot.html

Reply:
Cool thanks. Sounds good!